sdks/python/apache_beam/ml/inference/tensorrt_inference.py

```diff
@@ -0,0 +1,281 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# pytype: skip-file
+
+import logging
+import sys
+from typing import Any
+from typing import Dict
+from typing import Iterable
+from typing import Optional
+from typing import Sequence
+from typing import Tuple
+
+import numpy as np
+
+import tensorrt as trt
+from apache_beam.io.filesystems import FileSystems
+from apache_beam.ml.inference.base import ModelHandler
+from apache_beam.ml.inference.base import PredictionResult
+from cuda import cuda
```

Comment:
@azhurkevich Your original code imported cuda before tensorrt. Do you know if cuda has to be imported first? I reversed the order due to the `isort` command for the lint checks. However, if necessary, we can disable the sorting requirements here, and revert back to the previous order that you had.

Reply:
FWIW I think it should be ok to import CUDA after, but would be good to test that @azhurkevich

Reply:
@yeandy just tested, it is ok

Reply:
Ack. Thanks for testing!